modules/cms/classes/Page.php

```diff
@@ -112,11 +115,25 @@ public static function getNameList()
         return $result;
     }
 
+    /**
+     * Helper that returns formatted robots tags.
+     * @return string
+     */
+    public function getMetaRobotsAttribute()
+    {
+        $options = [];
+        $options[] = $this->meta_robots_noindex ? 'noindex' : 'index';
+        $options[] = $this->meta_robots_nofollow ? 'nofollow' : 'follow';
+        $options[] = $this->meta_robots_noarchive ? 'noarchive' : 'archive';
+        return implode(',', $options);
+    }
+
     /**
      * Helper that makes a URL for a page in the active theme.
      * @param mixed $page Specifies the Cms Page file name.
      * @param array $params Route parameters to consider in the URL.
      * @return string
+     * @throws CmsException
```

Comment:
Why have these been added?

Reply:
PHPStorm does this by himself. 

I'll clear that in the next commit.